.env-dist

```diff
@@ -2,3 +2,5 @@ DEBUG=True
 DEV=True
 ALLOWED_HOSTS=*
 SWITCH_NEWSLETTER_MAINTENANCE_MODE=False
+CSP_DEFAULT_SRC='self',*.mozilla.net,*.mozilla.org,*.mozilla.com,*.allizom.org
```

Comment:
When I run the site locally (not using Docker), this line seems to prevent all assets from loading? I also see things like `Content Security Policy: Couldn't parse invalid host self'` in the web console.

Things seems to be fine when using Docker, which is odd.

Reply:
@alexgibson I believe I've fixed this. It was an issue with the quotes around `'self'` and getting that out of the environment. Our library to do that strips quotes. So I moved to only neededing to provide the "extra" items to add to the `CSP_DEFAULT_SRC` list. This seems better anyway.